integration/kubernetes/multiMaster-embeddedJournal/alluxio-format-journal-job.yaml.template

```diff
@@ -32,10 +38,12 @@ spec:
               cpu: 1
               memory: 1G
           command: ["/opt/alluxio/bin/alluxio"]
-          args: ["formatMasters"]
+          args: ["fsadmin", "report"]
+          # To format journal, uncomment the below args and comment the above!
```

Comment:
Since we are going to change the way of creating these resources to `kubectl create -f dir/`, I don't want this job to be automatically created and wipe all the journal out. By making it run a report, the Job will just finish (or err) and die when the TTL past.

Reply:
this looks ugly. can we separate out the format job into a different directory instead? may the structure is `$dir/jobs/alluxio-format-journal-job`, `$dir/masters/...`, `$dir/workers/...`

Reply:
Ok now the resources are put under `$dir/master` `$dir/worker` and `$dir/job`, config is directly under `$dir` as it applies to all resources